Lightsaber NC handling issues

# Context

* KolMafia version: r20002
* Class: any
* Run type (Casual/Normal/Hardcore/Aftercore): any
* Challenge Path (Standard/Path Name/None): any
* Number of Ascensions: any
* Are you using the beta version of autoscend?: beta

# Current Behavior

When lightsaber calls on its NC, autoscend crashes instead of choosing an action. player must then choose an appropriate action and then run it again.

This happens to me fairly reliably in-run, doing softcore:

``> [INFO] - About to start a combat indirectly at Noob Cave... (2) accesses required.

cast 1 Calculate the Universe
numberology 51
Encounter: War Frat 151st Infantryman
Round 0: Jimmyton wins initiative!
Round 1: You lose 1 hit point
> [INFO] - autoAdvBypass has encountered a combat! (param: 'auto_combatHandler')

[236] Noob Cave
Encounter: War Frat 151st Infantryman
Round 0: Jimmyton loses initiative!
> [INFO] - auto_combatHandler: 0
Round 1: Jimmyton casts GULP LATTE!
Round 2: You gain 1 hit point
Round 2: You gain 954 Mana Points
Round 2: Ben nibbles your opponent for 7 damage, which sounds like a bad deal, but at least the extra-virulent bacteria are a free bonus.
Round 2: War Frat 151st Infantryman takes 7 damage.
Round 2: Your opponent writhes as he takes 7 damage from the poison.
Round 2: War Frat 151st Infantryman takes 7 damage.
Round 2: Jimmyton casts USE THE FORCE!
Encounter: Using the Force
Stack trace:

  at main (auto_post_adv.ash:1057)
> [ERROR] - Error running auto_post_adv.ash, setting auto_interrupt=true``

Happens reliably to me on War Frat 151st Infantryman and Burly Sidekick; possibly other monsters.

Sorry I don't have more logging here. What stuff do I need to set to get a useful log on my next run? And which logging, precisely, is useful? I can also insert logs into the ash scripts if there's something useful to know.

--- a/RELEASE/scripts/autoscend/iotms/mr2019.ash
+++ b/RELEASE/scripts/autoscend/iotms/mr2019.ash
@@ -350,18 +350,21 @@ int auto_saberChargesAvailable()
 string auto_combatSaberBanish()
 {
 	set_property("_auto_saberChoice", 1);
+	wait(3);
 	return "skill " + $skill[Use the Force];
 }
 
 string auto_combatSaberCopy()
 {
 	set_property("_auto_saberChoice", 2);
+	wait(3);
 	return "skill " + $skill[Use the Force];
 }
 
 string auto_combatSaberYR()
 {
 	set_property("_auto_saberChoice", 3);
+	wait(3);
 	return "skill " + $skill[Use the Force];
 }
 

